.travis.yml

```diff
@@ -15,6 +15,7 @@ script:
   - npm run docs:dist:gzip
   - test -e dist/index.html
   - test -e dist/sassdoc/index.html
+  - npm run update-bpk-dependencies
```

Comment:
I reckon this should be a lot earlier in the pipeline - even part of `npm test`?

Currently it'll only fail on CI.

Reply:
The reason I put it so late before is because it was such an expensive script. Now that it only takes a few seconds I agree 👍